draft-ietf-jsonpath-base.md

```diff
@@ -777,7 +780,7 @@ The following examples show the `wildcard` selector in use by a child segment.
 {: title="Wildcard selector examples"}
 
 The example above with the query `$.o[*, *]` shows that the wildcard selector may produce nodelists in distinct
-orders each time it appears in the child segment.
+orders each time it appears in the child segment, when it is applied to an object node (but not an array node).
```

Comment:
Fair enough. Do we also need to say that this is only for objects with more than one member? I'm a bit hesitant to expand on this non-normative text unduly.

Reply:
I like adding the "only one member" language but your call.

Reply:
Adopted in b590dcb9565a5cbcbf3f3d8990a2b2872a019090.